flask_ninja/operation.py

```diff
@@ -187,179 +158,200 @@ def serialize(cls, resp: Any) -> Any:
         """
         return json.loads(SerializationModel(data=resp).json())["data"]
 
-    def _obj_schema(self, obj: Any) -> dict:
-        """Generate schema for a model using pydantic and store definitions."""
-        schema = schema_of(obj, ref_template="#/components/schemas/{model}")
-        self.definitions.update(schema.get("definitions", {}))
-
-        if "definitions" in schema:
-            del schema["definitions"]
-
-        return schema
-
-    def get_callback_schema(self, cb: Callback) -> dict:
-        schema: dict = {}
+    def get_callback_schema(
+        self, cb: Callback, model_name_map: ModelNameMapType
+    ) -> dict[str, PathItem]:
         if cb.request_body:
-            schema["requestBody"] = {
-                "required": True,
-                "content": {
-                    "application/json": {"schema": self._obj_schema(cb.request_body)}
-                },
+            request_body, _, _ = field_schema(
+                create_model_field("Callback", cb.request_body),
+                model_name_map=model_name_map,
+                ref_prefix=REF_PREFIX,
+            )
+        else:
+            request_body = None
+
+        parameters: list[Union[Parameter, Reference]] = []
+        for param in cb.params or []:
+            field_info = param.field_info
+            field_info = cast(FuncParam, field_info)
+            if not field_info.include_in_schema:
+                continue
+            if field_info.in_ == ParamType.BODY:
+                continue
+            parameter = {
```

Comment:
with `allow_population_by_field_name = True` you can create `Parameter` model normally with constructor and use `in_` field name (see https://github.com/kiwicom/flask-ninja/pull/14/files#r1150475198)

Reply:
Thanks, that was one issue, and there was another issue why I decided to construct dict
mypy complained about generic types - example provided `list[int]` but expected is `list[Union[str, int]]`. But after I upgraded mypy to v1.1.1, this issue is solved there. Now everywhere I use objects, not dicts